### PR TITLE
[Vercel] Update gunman.json

### PR DESCRIPTION
**Pull request by MukuJhansi**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.
Changing host to Vercel 
Please note that we reserve the rights not to accept any domain at our own discretion.

- [X] The file is in the `domains` folder and is in the JSON format.
- [X] The file's name is all lowercased and alphanumeric. 
- [X] You have completed your website. 
- [X] The website is reachable.  
- [X] The CNAME record doesn't contain `https://` or `/`.  
- [X] There is sufficient information at the `owner` field.  

## Website Link/Preview

https://main-mukujhansis-projects.vercel.app

### CODE REVIEW

File: domains/gunman.json [hunk 1]
```diff
@@ -5,6 +5,6 @@
     "discord": "1115658967012626542"
   },
   "record": {
-    "A": ["216.24.57.1"]
+    "A": ["76.76.21.21"]
```
Comment (Stef-00012) [inline domains/gunman.json:8]: Your website doesn't seem an incomplete template, i see "lorem ipsum" strings

**Review by Stef-DP — CHANGES_REQUESTED** [review]

File: domains/gunman.json (hunk 1 above)
Comment (MukuJhansi) [reply domains/gunman.json:8]: I am currently developing it and I need a text to fill it so I used lorem ipsum.

**Comment by MukuJhansi** [comment]
I have a another one website too, and the website is not made by a template but instead I used the lorem<no of word> shortcut of vscode to fill empty space

**Comment by MukuJhansi** [comment]
396483749-ff5f929d-f2a8-45ac-b1be-0920c201a196

**Comment by iostpa** [comment]
I'd recommend replacing the Lorem ipsum text with stuff about you, then I might merge it.

**Pull request closed without merging**